modules/createHashHistory.js

```diff
@@ -33,7 +37,7 @@ var DefaultQueryKey = '_k';
 function createHashHistory(options={}) {
   var { queryKey } = options;
 
-  if (queryKey && typeof queryKey !== 'string')
+  if (typeof queryKey !== 'string')
```

Comment:
But MemoryHistory and BrowserHistory are saving state by default, should HashHistory be saving it too? In that case maybe pass false as query key to disable saving state? `createHashHistory({ queryKey: false })` or something like that?